.travis.yml

```diff
@@ -1,38 +1,35 @@
 language: rust
 rust:
-  - 1.36.0 # minimum supported toolchain & pinned toolchain for clippy
-  - stable
+  - 1.42.0 # minimum supported toolchain
+  - 1.49.0 # pinned toolchain for clippy
```

Comment:
I think the problem with running clippy on anything but the min. supported version is that it might suggests backwards incompatible changes. wdyt?